Update VTOL Guide

itscynxx opened the pull request:
Changes some wording of the guide, adds and updates some images

I actually thought i was gonna have a _lot_ more work, but I guess not, might add sections about plugins and dependencies installing later but this is just meant to maintain it and update some information that was just lacking before

BigSpice commented:
tysm, im back from work this weekend, so im pushing 3.0 then :)

BigSpice reviewed (approved):
Wonderful work

GeckoEidechse commented on docs/images/vtol-settings.png:
I wouldn't update screenshots just for the version number. So unless there's bigger changes to the UI I'd keep the old ones. Otherwise it just blows up the repo in size over time ^^"

GeckoEidechse commented on docs/installing-northstar/northstar-installers/vtol-guide.md:
```diff
@@ -1,31 +1,37 @@
-VTOL is a mod manager for installing and managing mods for the Northstar client made for Titanfall 2. You can find links to install it on the [Northstar website](https://northstar.tf), or on [VTOL's GitHub page](https://github.com/R2NorthstarTools/VTOL).
+VTOL is a mod manager for installing and managing mods for the Northstar client made for Titanfall 2.
+
+VTOL works on Windows only, and you can find links to download it on [Northstar's website](https://northstar.tf) or [VTOL's GitHub repo](https://github.com/R2NorthstarTools/VTOL). Installing it from the website automatically installs the latest version that will automatically update itself. On the GithHubn repo, you can download the newest version by clicking one of two optionjs. Note that the po ...
```
Newlines in source for better diff + fixed typos ^^
```suggestion
VTOL works on Windows only, and you can find links to download it on [Northstar's website](https://northstar.tf) or [VTOL's GitHub repo](https://github.com/R2NorthstarTools/VTOL).
Installing it from the website automatically installs the latest version that will automatically update itself.
On the GithHub repo, you can download the newest version by clicking one of two options.
Note that the portable is just a `.exe` file, and not the installer, and as such it won't automatically update itself.
```

GeckoEidechse commented on docs/installing-northstar/northstar-installers/vtol-guide.md:
```diff
@@ -1,31 +1,37 @@
+
+![VTOL installer download (on GitHub, automatically updates](../../images/vtol-download-github.png)
+
+![VTOL portable download (on GitHub, doesn't automnatically update](../../images/vtol-download-github-portable.png)
```
```suggestion
![VTOL portable download (on GitHub, doesn't automatically update](../../images/vtol-download-github-portable.png)
```

itscynxx commented:
i removed some unnecessary image changes (i kept the ones where the ui looks different now, mostly the change of `Update +Northstar` to `Re-install Northstar`), changed some wording that was just bad, fixed a couple dumb typos, and added some new lines for reading

GeckoEidechse commented on docs/installing-northstar/northstar-installers/vtol-guide.md:
```diff
@@ -1,31 +1,39 @@
+VTOL is a mod manager for installing and managing mods for the Northstar client made for Titanfall 2.
+
+VTOL works on Windows only, and you can find links to download it on [Northstar's website](https://northstar.tf) or [VTOL's GitHub repo](https://github.com/R2NorthstarTools/VTOL).
+Installing it from the website automatically installs the latest version that will automatically update itself. On the GithHubn repo, you can download the newest version by clicking one of two options. 
```
Typo
```suggestion
Installing it from the website automatically installs the latest version that will automatically update itself. On the GitHub repo, you can download the newest version by clicking one of two options. 
```

GeckoEidechse commented on docs/installing-northstar/northstar-installers/vtol-guide.md:
```diff
@@ -1,31 +1,39 @@
 ## Version Numbers
 
-You can see what _"version of VTOL/Northstar"_ you have in the top left corner of the VTOL window.
+You can see what version of VTOL/Northstar you have in the top left corner of the VTOL window.
```
Which one is it, VTOL or Northstar? :P
```suggestion
In the top left corner of the VTOL window you can see what versions of VTOL and Northstar you have installed.
```

GeckoEidechse commented on docs/installing-northstar/northstar-installers/vtol-guide.md:
```diff
@@ -72,27 +83,33 @@ You can also right click on a specific mod to get the option to either delete it
 ![Skins installation tab](../../images/vtol-skins.png)
 
-The third settings tab on VTOL is the _"Skins"_ tab. VTOL has support for skins formatted for an older version of Northstar, normally requiring an extra tool. All you need to do is drag the folder of one of these old style skins onto the Skins screen to install it automatically.
+The third settings tab on VTOL is the _"Skins"_ tab. VTOL has support for DDS format skins, which are commonly used for Vanilla Titanfall 2, with Northstar generally opting to convert them to mods.
```
I would still mention that DDS skins are deprecated.

Like you can use them for vanilla but you have to overwrite files. You could also just use Northstar on vanilla to use the new skin format :P

GeckoEidechse commented on docs/installing-northstar/northstar-installers/vtol-guide.md:
```diff
@@ -72,27 +83,33 @@ You can also right click on a specific mod to get the option to either delete it
 
-The fourth tab on VTOL is the _"Server"_ tab. This section allows you to add Launch Arguments to your client or dedicated server. Note that this adds Launch Arguments to change what Northstar does when it launches, not to launch Northstar itself. A way you can do this is by adding [Launch Arguments](../troubleshooting.md#launch-opts) via the store you own the game on and launching through there.
+The fourth tab on VTOL is the _"Server"_ tab. This section allows you to add Launch Arguments to your client or dedicated server.
+Note that this adds Launch Arguments to change what Northstar does when it launches, not to launch Northstar itself.
```
The second part of the sentence actually kinda confused me more.

It's just standard launch arg stuff, right?

GeckoEidechse commented on docs/installing-northstar/northstar-installers/vtol-guide.md:
```diff
@@ -112,13 +129,14 @@ The first screen is an area where you can locate a folder for a mod you've creat
 
-The third screen is an implementation of Spoon's [Advocate Tool](https://github.com/ASpoonPlaysGames/Advocate), where you can convert an old style skin mod to a mod that can be installed by everyone like a normal mod. Please give credit to the original skin owner while doing this. (If confused, following the earlier link to Github describes how to set up Advocate)
+The third screen is an implementation of Spoon's [Advocate Tool](https://github.com/ASpoonPlaysGames/Advocate), where you can convert a DDS formatted skin to a mod that can be installed by everyone normally without the need for the [Skin Tool](https://github.com/zxcPandora/Titanfall2-SkinTool).
+Please give credit to the original skin owner while doing this. (If confused on how to use this, follow the earlier link to Github describes how to set up Advocate. Do note, this version of Advocate isn't perfectly up to date)
```
This is kinda confusing. There's only a need to give credit when you actually share the skin. You don''t need to do anything if you just use it for yourself.

This kinda makes it sounds like you have to say "thank you skin creator" everytime you launch the game xd

GeckoEidechse commented:
Any updates on this?

itscynxx commented:
> Any updates on this?

i forgor 💀

GeckoEidechse commented on docs/installing-northstar/northstar-installers/vtol-guide.md:
```diff
@@ -1,31 +1,39 @@
-VTOL is a mod manager for installing and managing mods for the Northstar client made for Titanfall 2. You can find links to install it on the [Northstar website](https://northstar.tf), or on [VTOL's GitHub page](https://github.com/R2NorthstarTools/VTOL).
+VTOL is a mod manager for installing and managing mods for the Northstar client made for Titanfall 2.
+
+VTOL works on Windows only, and you can find links to download it on [Northstar's website](https://northstar.tf) or [VTOL's GitHub repo](https://github.com/R2NorthstarTools/VTOL).
```
```suggestion
VTOL works on Windows only and you can find links to download it on [Northstar's website](https://northstar.tf) or [VTOL's GitHub repo](https://github.com/R2NorthstarTools/VTOL).
```

GeckoEidechse commented on docs/installing-northstar/northstar-installers/vtol-guide.md:
```diff
@@ -1,31 +1,39 @@
+
+VTOL works on Windows only, and you can find links to download it on [Northstar's website](https://northstar.tf) or [VTOL's GitHub repo](https://github.com/R2NorthstarTools/VTOL).
+Installing it from the website automatically installs the latest version that will automatically update itself. On the GitHub repo, you can download the newest version by clicking one of two options. 
+Note that the portable is just a `.exe` file, and not the installer, and as such it won't automatically update itself.
```
```suggestion
Note that the portable is just a `.exe` file and not the installer, as such it won't automatically update itself.
```

GeckoEidechse commented on docs/installing-northstar/northstar-installers/vtol-guide.md:
```diff
@@ -37,25 +45,28 @@ VTOL handles most things automatically, but it may still require minimal setup f
 
-After doing this, Northstar should automatically install. If not, the button at the bottom will read _"Install Northstar"_. Let VTOL install Northstar, and you should be good to go. Once installed, this button will turn into the _"Update +Northstar"_ shown in the screenshot earlier, which you can use to update without having to reinstall.
+After doing this, Northstar should automatically install. If not, the button at the bottom will read _"Install Northstar"_. 
+Let VTOL install Northstar, and you should be good to go. Once installed, this button will turn into the _"Re-Install Northstar"_ button shown in the screenshot earlier, which you can use to forcibly reinstall Northstar in the case something goes wrong with your Northstar install.
```
```suggestion
Let VTOL install Northstar and you should be good to go. Once installed, this button will turn into the _"Re-Install Northstar"_ button shown in the screenshot earlier, which you can use to forcibly reinstall Northstar in the case something goes wrong with your Northstar install.
```

GeckoEidechse commented on docs/installing-northstar/northstar-installers/vtol-guide.md:
```diff
@@ -72,27 +83,34 @@ You can also right click on a specific mod to get the option to either delete it
-The third settings tab on VTOL is the _"Skins"_ tab. VTOL has support for skins formatted for an older version of Northstar, normally requiring an extra tool. All you need to do is drag the folder of one of these old style skins onto the Skins screen to install it automatically.
+The third settings tab on VTOL is the _"Skins"_ tab. VTOL has support for DDS format skins (the skins with the numbered folders in them), which are commonly used for Vanilla Titanfall 2, with Northstar generally opting to convert them to mods.
+
+This format is older, and less consistent than converting them to a mod to be used with Northstar (see [Advocate](./#advocate)), and can still be used on Vanilla by making use of a [vanilla profile](../using-northstar/advanced.md#vanilla-profile).
```
```suggestion
This format is older and less consistent than converting them to a mod to be used with Northstar (see [Advocate](./#advocate)). It can still be used on Vanilla by making use of a [vanilla profile](../using-northstar/advanced.md#vanilla-profile).
```

GeckoEidechse commented on docs/installing-northstar/northstar-installers/vtol-guide.md:
```diff
@@ -130,11 +149,13 @@ The fourth screen lists a lot of commonly used tools by modders to help create m
 
-The sixth tab is the _"Profiles"_ tab, allowing you to create profiles for Northstar. Profiles are seperate Northstar installs, each with their own set of mods and seperated by folders. For example, you can have one profile with your normal set of mods, and another profile for only using core mods (those that common with Northstar/the ones that start with `Northstar.`).
+The sixth tab is the _"Profiles"_ tab, allowing you to create profiles for Northstar. Profiles are seperate Northstar installs, each with their own set of mods and seperated by folders. 
+For example, you can have one profile with your normal set of mods, and another profile for only using core mods (those that common with Northstar/the ones that start with `Northstar.`).
```
```suggestion
For example, you can have one profile with your normal set of mods and another profile for only using core mods (those that common with Northstar/the ones that start with `Northstar.`).
```

GeckoEidechse reviewed (commented):
Grammar and punctuation
